onnxscript/evaluator.py

```diff
@@ -483,6 +501,18 @@ def _eval(self, schema, inputs, attributes, closure):
         return _call_ort(schema, inputs, attributes, closure)
 
 
+class OnnxReferenceRuntimeEvaluator(BaseEvaluator):
+    """Evaluates ONNX ops using ONNX Runtime."""
+
+    def _eval(self, schema, inputs, attributes, closure):
+        model, session_run_input, _ = _prepare_model_and_inputs_for_eager(
+            schema, inputs, attributes, closure
+        )
+        session = onnx.reference.ReferenceEvaluator(model)
+        result = session.run(None, session_run_input)
+        return [_ort_to_os_value(x) for x in result]
```

Comment:
rename function? ort is unrelated under this call.

Is it just happen so both onnx reference runtime and ort are using same value encoding? Please document.

Reply:
Updated to _numpy_to_onnxscript_value